src/net/sourceforge/zmanim/ComplexZmanimCalendar.java

```diff
@@ -89,18 +89,39 @@
  */
 public class ComplexZmanimCalendar extends ZmanimCalendar {
 
+    public static final int SHAAH_ZMANIS_120MINUTES = 2;
```

Comment:
Any reason not to use an enum instead of a set of "accidentally" related constants?

(Same for the constants in the ZmanimCalendar class.)